src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -422,21 +405,17 @@ export const useSwapper = () => {
         amount,
         swapper,
         sellAsset,
-        feeAsset,
         buyAsset,
         action,
         wallet,
         accountSpecifiersList,
         selectedCurrencyToUsdRate,
         sellAssetAccount,
+        sellAssetUsdRate,
+        buyAssetUsdRate,
+        feeAssetUsdRate,
       }: DebouncedQuoteInput) => {
         try {
-          const [sellAssetUsdRate, buyAssetUsdRate, feeAssetUsdRate] = await Promise.all([
-            swapper.getUsdRate({ ...sellAsset }),
-            swapper.getUsdRate({ ...buyAsset }),
-            swapper.getUsdRate({ ...feeAsset }),
-          ])
```

Comment:
This removal is the meat of this PR. This logic is now handled by RTK Query at the beginning of `useSwapper`.